docs/Overview.md

```diff
@@ -13,25 +13,6 @@ Accessibility Insights takes the data provided by UIA and compares it to a set o
 ### Code Organization
 The code is organized into the following general areas:
 
-#### Runtime components
-These assemblies provide the interaction with UIA, as well as layers that allow the application to interact with that information in a more structured manner:
-
-Assembly | Responsibility
---- | ---
-Axe.Windows.Actions | Provide a high-level set of Actions that are the primary interface into the Runtime components.
-Axe.Windows.Core | Provide data abstractions which represent accessibility data in a platform-agnostic way.
-Axe.Windows.Desktop | Provide platform-specific (Windows) implementations of the platform-agnostic data abstractions. The low-level interactions with UIA occur in this assembly.
-Axe.Windows.Telemetry | Provides an interface which any caller can provide to capture telemetry from Axe.Windows
-AccessibilityInsights.Win32 | Provide a wrapper around Win32-specific code that is needed by other assemblies.
```

Comment:
Don't we still have AccessibilityInsights.Win32?

Reply:
Oops! Re-added in iteration 8

---
In reply to: [290824275](https://github.com/microsoft/accessibility-insights-windows/pull/385#discussion_r290824275) [](ancestors = 290824275)